content/en/blog/_posts/2022-12-XX-kubernetes-release-artifact-signing.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: blog
+title: "Kubernetes 1.26: We're now signing our binary release artifacts!"
+date: 2022-12-6
+slug: kubernetes-release-artifact-signing
+---
+
+**Author:** Sascha Grunert
+
+The Kubernetes Special Interest Group (SIG) Release is proud to announce that we
+graduated the [Signing Release Artifacts Enhancement Proposal][kep] to _beta_.
+
+[kep]: https://github.com/kubernetes/enhancements/issues/3031
```

Comment:
```suggestion
are digitally signing all release artifacts, and that this aspect of Kubernetes has now reached
_beta_.
```

KEPs are usually footnotes. We want to signpost readers to docs, unless we're writing for the contributor blog.

Reply:
Moved it to the bottom to an "Additional resources" section